models/stg_reddit_ads__account_report.sql

```diff
@@ -30,11 +29,11 @@ final as (
         conversion_roas,
         cpc,
         ctr,
-        date_day, -- renamed in macro
+        date_day, -- renamed in macro 
         ecpm,
         impressions,
         region,
-        (spend/1000000) as spend,
+        (spend/1000000) as spend, 
```

Comment:
Do you have a link to the Reddit Ads documentation that states this is in microcurrency amounts? If so, can we link it within the docs.

Reply:
The definition for spend per the reddit docs is noted in microcurrency. And this is noted in the yml docs. I update the yml doc with the link

https://ads-api.reddit.com/docs/#tag/Reporting/paths/~1api~1v2.0~1accounts~1{account_id}~1reports/get

![image](https://user-images.githubusercontent.com/91097070/222003136-47853ccc-0f14-423f-9561-f79d0e182d96.png)